### PR TITLE
add support version of exif to properly read rotation

### PR DESCRIPTION
**Pull request by ebbnormal**
this change uses the support version of the [exifinterface](https://android-developers.googleblog.com/2016/12/introducing-the-exifinterface-support-library.html) library and uses it to properly read the rotation tags on JPEG and finally rotates the image if necessary before writing to the output stream. @acrimi if this could be expedited for the purposes of it being for the sake of the _Krankenschwestere_ of the _Vereinigtestaaten._

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:26.1.0'
+    compile 'com.android.support:exifinterface:26.1.0'
```
Comment (acrimi) [inline core/build.gradle:25]: Do we not need appcompat?

File: core/src/main/java/com/isbx/androidtools/media/ImageResizer.java [hunk 2]
```diff
@@ -191,6 +191,27 @@ public boolean imageIsJPEG(Uri imageUri) throws Exception {
         }
     };
 
+    public Bitmap rotateImage(Uri imageUri, Bitmap sourceImage) throws IOException {
+        Bitmap bitmap = sourceImage;
+        ExifInterface exif = new ExifInterface(context.getContentResolver().openInputStream(imageUri));
+        int rotation = exif.getAttributeInt(ExifInterface.TAG_ORIENTATION, 
+                                            ExifInterface.ORIENTATION_NORMAL);
+        int rotationInDegrees = exifToDegrees(rotation);
+        Log.i("DEV", String.valueOf(rotationInDegrees) + " degrees rotated");
```
Comment (acrimi) [inline core/src/main/java/com/isbx/androidtools/media/ImageResizer.java]: please remove log

File: databinding/build.gradle [hunk 3]
```diff
@@ -26,5 +26,5 @@ android {
 }
 
 dependencies {
-    compile 'com.github.acrimi.AndroidUtils:core:core_0.0.14'
-}
\ No newline at end of file
+    compile 'com.github.ebbnormal.AndroidUtils:core:core_0.0.26'
```
Comment (acrimi) [inline databinding/build.gradle]: Revert dependency

File: location/build.gradle [hunk 4]
```diff
@@ -22,7 +22,7 @@ android {
 }
 
 dependencies {
-    compile 'com.github.acrimi.AndroidUtils:core:core_0.0.14'
+    compile 'com.github.ebbnormal.AndroidUtils:core:core_0.0.26'
```
Comment (acrimi) [inline location/build.gradle]: revert dependency

**Review by acrimi — CHANGES_REQUESTED** [review]

File: core/build.gradle (hunk 1 above)
Comment (ebbnormal) [reply core/build.gradle:25]: Do we not?

Comment (ebbnormal) [reply core/build.gradle:25]: When do we use it tho

File: databinding/build.gradle (hunk 3 above)
Comment (ebbnormal) [reply databinding/build.gradle]: so how will this work, i just point to the old dependencies, and then you will slowly create tags of each one starting with core (Which has the changes), and then update the tags with the new tag you make?

Comment (acrimi) [reply databinding/build.gradle]: yes, dependencies are updated when new releases are created

File: core/build.gradle (hunk 1 above)
Comment (acrimi) [reply core/build.gradle:25]: guess not

**Review by acrimi — APPROVED** [review]

**Pull request merged**